modules/rostests/winetests/wtsapi32/CMakeLists.txt

```
@@ -1,6 +1,7 @@
 
+add_definitions(-D__WINESRC__)
 add_definitions(-DUSE_WINE_TODOS)
```

```suggestion
add_definitions(-D__WINESRC__ -DUSE_WINE_TODOS)
remove_definitions(-D__ROS_LONG64__)

```
See next comments regarding `__ROS_LONG64__`